src/cpu/cpu.rs

```diff
@@ -70,1165 +70,1140 @@ impl Cpu {
-                2 + result as u8 // + page boundary crossed
+                cyc_inc_by = 2 + result as u8 // + page boundary crossed;
             }
 
             0x11 => {
-                let data = cpu_map.get(self.pc + 1);
-                let (opop, add_cycle) = izy(cpu_map, data, self.y);
-                ora(self, cpu_map, opop);
+                let operand = cpu_map.get(self.pc + 1);
+                let (addr, add_cycle) = izy(operand, self.y, cpu_map);
+                ora(cpu_map.get(addr), self);
                 pc_inc_by = 2;
-                5 + add_cycle as u8
+                cyc_inc_by = 5 + add_cycle as u8;
             }
 
             0x15 => {
-                let data = cpu_map.get(self.pc + 1);
-                let opop = zpx(data, self.x);
-                ora(self, cpu_map, opop);
+                let operand = cpu_map.get(self.pc + 1);
+                let addr = zpx(operand, self.x);
+                ora(cpu_map.get(addr), self);
                 pc_inc_by = 2;
-                4
+                cyc_inc_by = 4;
             }
 
             0x16 => {
-                let data = cpu_map.get(self.pc + 1);
-                let opop = zpx(data, self.x);
-                asl(self, cpu_map, opop);
+                let operand = cpu_map.get(self.pc + 1);
+                let addr = zpx(operand, self.x);
+                asl(addr, self, cpu_map);
                 pc_inc_by = 2;
-                4
+                cyc_inc_by = 4;
             }
 
             0x18 => {
                 clc(self);
                 pc_inc_by = 1;
-                2
+                cyc_inc_by = 2;
             }
 
             0x19 => {
-                let data = cpu_map.get_16(self.pc + 1);
-                let (opop, add_cycle) = aby(data, self.y);
-                ora(self, cpu_map, opop);
+                let operand = cpu_map.get_16(self.pc + 1);
+                let (addr, add_cycle) = aby(operand, self.y);
+                ora(cpu_map.get(addr), self);
                 pc_inc_by = 3;
-                4 + add_cycle as u8
+                cyc_inc_by = 4 + add_cycle as u8;
             }
 
             0x1D => {
-                let data = cpu_map.get_16(self.pc + 1);
-                let (opop, add_cycle) = abx(data, self.x);
-                ora(self, cpu_map, opop);
+                let operand = cpu_map.get_16(self.pc + 1);
+                let (addr, add_cycle) = abx(operand, self.x);
+                ora(cpu_map.get(addr), self);
                 pc_inc_by = 3;
-                4 + add_cycle as u8
+                cyc_inc_by = 4 + add_cycle as u8;
             }
 
             0x1E => {
-                let data = cpu_map.get_16(self.pc + 1);
-                let (opop, add_cycle) = abx(data, self.x);
-                asl(self, cpu_map, opop);
+                let operand = cpu_map.get_16(self.pc + 1);
+                let (addr, add_cycle) = abx(operand, self.x);
+                asl(addr, self, cpu_map);
```

Comment:
It's weird for `self` to not be the first parameter...

I mean it's also weird that these are functions and not methods, we should probably fix that too...